libtextsecure/protobufs.js

```diff
@@ -4,15 +4,11 @@
     window.textsecure.protobuf = {};
 
     function loadProtoBufs(filename) {
-        return dcodeIO.ProtoBuf.loadProtoFile({root: 'protos', file: filename}, function(error, result) {
-           if (error) {
-              throw error;
-           }
-           var protos = result.build('textsecure');
-           for (var protoName in protos) {
-              textsecure.protobuf[protoName] = protos[protoName];
-           }
-        });
+        var result = dcodeIO.ProtoBuf.loadProtoFile({root: 'protos', file: filename});
```

Comment:
Just flagging this for consideration, btw, pretty sure this will work fine for now at least...